agent/command/git_test.go

```diff
@@ -363,7 +363,7 @@ func (s *GitGetProjectSuite) TestGitFetchRetries() {
 	c := gitFetchProject{Directory: "dir"}
 
 	conf := s.taskConfig1
-	conf.Distro.CloneMethod = "this is not real!"
+	conf.Distro = &apimodels.DistroView{}
```

Comment:
this test fails with nil pointer dereference to a jasper function that I don't know why is panicking. Would appreciate someone to help me look at it :(

Reply:
I think this just needs to call `c.SetJasperManager(s.jasper)` since there's no default Jasper implementation for the command on L363.

I'm also not really sure what this test case is testing since I don't see how it's testing retries, but that's an unrelated issue 🤷 Just setting the Jasper manager should make it pass.

Reply:
I guess they were using a fake distro to fail and retry so they never ran into the jasper error in the past